src/pages/add-lock-exception/AddLockExceptionForm/PeriodPicker.js

```diff
@@ -1,19 +1,76 @@
-import { is53WeekISOYear, getFirstDateOfWeek } from 'd2/period/helpers'
+import i18n from '@dhis2/d2-i18n'
+import { SingleSelectField, SingleSelectOption } from '@dhis2/ui'
 import DatePicker from 'material-ui/DatePicker'
-import MenuItem from 'material-ui/MenuItem'
-import SelectField from 'material-ui/SelectField'
 import PropTypes from 'prop-types'
 import React from 'react'
 
+const getFirstDateOfWeek = (year, week) => {
+    const ordTable = [0, 31, 59, 90, 120, 151, 181, 212, 243, 273, 304, 334]
+    const ordTableLeap = [0, 31, 60, 91, 121, 152, 182, 213, 244, 274, 305, 335]
+    const isLeapYear =
+        new Date(new Date(year, 2, 1).setDate(0)).getDate() === 29
+    const ordDiff = isLeapYear ? ordTableLeap : ordTable
+
+    const correction = (new Date(year, 0, 4).getDay() || 7) + 3
+    const ordDate = week * 7 + (1 - correction)
+    if (ordDate < 0) {
+        return new Date(year, 0, ordDate)
+    }
+
+    let month = 11
+    while (ordDate < ordDiff[month]) {
+        month--
+    }
+
+    return new Date(year, month, ordDate - ordDiff[month])
+}
+
+const is53WeekISOYear = year => {
+    const p = y =>
+        y + Math.floor(y / 4) - Math.floor(y / 100) + Math.floor(y / 400)
+
+    return p(year) % 7 === 4 || p(year - 1) % 7 === 3
+}
+
+const periodTypeLabels = {
+    week: i18n.t('week'),
+    month: i18n.t('month'),
+    year: i18n.t('year'),
+    biWeek: i18n.t('bi weekly'),
+    biMonth: i18n.t('bi monthly'),
+    day: i18n.t('day'),
+    jan: i18n.t('jan'),
+    feb: i18n.t('feb'),
+    mar: i18n.t('mar'),
+    apr: i18n.t('apr'),
+    may: i18n.t('may'),
+    jun: i18n.t('jun'),
+    jul: i18n.t('jul'),
+    aug: i18n.t('aug'),
+    sep: i18n.t('sep'),
+    oct: i18n.t('oct'),
+    nov: i18n.t('nov'),
+    dec: i18n.t('dec'),
+    'jan-feb': i18n.t('jan-feb'),
+    'mar-apr': i18n.t('mar-apr'),
+    'may-jun': i18n.t('may-jun'),
+    'jul-aug': i18n.t('jul-aug'),
+    'sep-oct': i18n.t('sep-oct'),
+    'nov-dec': i18n.t('nov-dec'),
+    quarter: i18n.t('quarter'),
+    Q1: i18n.t('Q1'),
+    Q2: i18n.t('Q2'),
+    Q3: i18n.t('Q3'),
+    Q4: i18n.t('Q4'),
+    sixMonth: i18n.t('six monthly'),
+    'jan-jun': i18n.t('jan-jun'),
+    'jul-dec': i18n.t('jul-dec'),
+    'apr-sep': i18n.t('apr-sep'),
+    'oct-mar': i18n.t('oct-mar'),
+}
```

Comment:
These labels were previously stored in [`AddLockExceptionForm.js`](https://github.com/dhis2/data-administration-app/blob/master/src/pages/lock-exception/AddLockExceptionForm.js#L54-L102)